u_common.go

```diff
@@ -155,6 +160,20 @@ var (
 	HelloIOS_Auto = HelloIOS_12_1
 	HelloIOS_11_1 = ClientHelloID{helloIOS, "111", nil} // legacy "111" means 11.1
 	HelloIOS_12_1 = ClientHelloID{helloIOS, "12.1", nil}
+
+	HelloEdge_Auto = HelloEdge_85
+	HelloEdge_85   = ClientHelloID{helloEdge, "85", nil}
+
+	HelloExplorer_11 = ClientHelloID{helloExplorer, "11", nil}
+
+	HelloSafari_Auto = HelloSafari_13_1
+	HelloSafari_13_1 = ClientHelloID{helloSafari, "13.1", nil}
```

Comment:
I specified the minor version for Safari because iOS is specified by the minor version as well.  I figured Safari's release cycle may be more similar to that of iOS than Chrome or Firefox.

Reply:
Works for me :+1: